docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -241,19 +241,44 @@ In order to use a Nessie catalog, ensure to configure the catalog type with
 properties:
 
 .. list-table:: Nessie catalog configuration properties
-  :widths: 40, 60
+  :widths: 30, 58, 12
```

Comment:
Since a bunch of parameters do not have default values its better to stick with two columns. Then add the default value in the description .. see later comments.